### ISSUE TITLE
Use Zypak to keep sandboxing enabled in flatpak

### ISSUE THREAD

**Issue by rugk** [flathub/com.vscodium.codium#55]
> FWIW, [com.visualstudio.code](https://github.com/flathub/com.visualstudio.code/pull/237) and [com.visualstudio.code-oss](https://github.com/flathub/com.visualstudio.code-oss/tree/zypak) are working on solutions with [Zypak](https://github.com/refi64/zypak).  (I don't know much about flatpak or anything about Zypak; just sharing information that turned up in the course of researching this problem.)

_Originally posted by @yump in https://github.com/flathub/com.vscodium.codium/issues/50#issuecomment-886417911_

---

I just wanted to open a new issue for that as I think it is a good idea (and as you can see, other's are working on it), and I think it would also be useful for the VsCodium flatpak.

**Comment by noonsleeper** [comment]
Yeah! sounds good

**Comment by dada513** [comment]
Any progress on this? If not, I can submit a PR

**Comment by dada513** [comment]
blocked by https://github.com/flathub/ide-flatpak-wrapper/issues/11 it seems

**Comment by TheEvilSkeleton** [comment]
It looks like the VS Code Flatpak uses Zypak now:

https://github.com/flathub/com.visualstudio.code/blob/7c97ff0dc0ad185bf5268c64c9d623de1784c64a/com.visualstudio.code.yaml#L137-L142

**Comment by rugk** [comment]
Any news here? What prevents you from just copying the VSCode-way?

**Comment by noonsleeper** [comment]
I tried, and doesn't work at least the 3 ways that I tried, feel free to make a PR if you can accomplish this @rugk.

**Comment by rugk** [comment]
Okay, hmm… I am too unexperienced here, so I cannot help. However, maybe @catsout can help [who seem to have introduced that feature in VSCode](https://github.com/flathub/com.visualstudio.code/commit/50940d894d9943d8b209fddc3e6651ab25968f5d) in https://github.com/flathub/com.visualstudio.code/pull/300.
